### PR TITLE
[dv:entropy_src] Updates to testplan, checklist

### PR DESCRIPTION
**Pull request by senelson7**
Signed-off-by: Steve Nelson <steve.nelson@wdc.com>

### CODE REVIEW

File: hw/ip/entropy_src/data/entropy_src_testplan.hjson [hunk 1]
```diff
@@ -9,9 +9,83 @@
...
-      desc: '''Enable entropy_src. Verify entropy, status, interrupt bits.'''
+      desc: '''Enable entropy_src in LFSR mode, wait for interrupt, verify entropy.'''
       milestone: V1
       tests: ["entropy_src_sanity"]
     }
+    {
+      name: efuse
+      desc: '''Verify entropy is not available via register if efuse input is 0.'''
+      milestone: V2
+      tests: [""]
```
Comment (sriyerg) [inline hw/ip/entropy_src/data/entropy_src_testplan.hjson:20]: nit: leave the `tests` key as an empty list `[]` as opposed to a list with single empty string entry.

File: hw/ip/entropy_src/data/entropy_src_testplan.hjson [hunk 2]
```diff
@@ -9,9 +9,83 @@
...
+    }
+    {
+      name: health_check_rct
+      desc: '''Verify RCT health check for various cutoff values.'''
+      milestone: V2
+      tests: [""]
+    }
+    {
+      name: stress
```
Comment (sriyerg) [inline hw/ip/entropy_src/data/entropy_src_testplan.hjson:77]: We have added infrastructure to construct `stress_all` and `stress_all_with_rand_reset` tests. 
The former runs all individual test sequences sequentially, while injecting TL error and CSR / MEM test sequences in parallel (which should map to this entry). The latter goes one step beyond by randomly asserting reset in the middle of the test. 
Consider adding the latter one as well.

**Comment by senelson7** [comment]
Addressed Sri's comments...

**Comment by senelson7** [comment]
Is there anything holding up this PR, it's been a couple weeks? It just updates the checklist and testplan. Once it's merged, I'd like to do the V1 review unless you'd like to do that prior to merging.

File: hw/ip/entropy_src/data/entropy_src_testplan.hjson [hunk 3]
```diff
@@ -9,9 +9,90 @@
...
+      name: stress_all
+      desc: '''
+            This will combine the other individual testpoints while injecting TL errors and running CSR tests in parallel.'''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: stress_all_with_random_reset
+      desc: '''
+            This will combine the other individual testpoints while injecting TL errors and running CSR tests in parallel and asserting reset in the middle of test.'''
```
Comment (sriyerg) [inline hw/ip/entropy_src/data/entropy_src_testplan.hjson:93]: Consider wrapping lines at 100 characters.

**Pull request merged**